GeometryCore/src/Polygon.cpp

```diff
@@ -252,4 +252,83 @@ namespace geom_utils
         return poly;
     }
 
+    inline FPoint2D vecRot90CW(const FPoint2D& point) 
+    {
+        FPoint2D newPoint(point.y,-point.x);
+        return newPoint;
+    }
+
+    inline FPoint2D vecRot90CCW(const FPoint2D& point) 
```

Comment:
Common function, so I suggest to move it out of Polygon.cpp and place in linear algebra